src/main/resources/scripts/-examples/experimental features/script reflection.sk

```diff
@@ -0,0 +1,245 @@
+using examples
+
+# This flag enables the experimental 'script reflection' feature within this file
+# Script reflection is a collection of powerful tools for scripts to inspect themselves
+using script reflection
+
+
+###
+THE BASICS
+=====
+
+'Reflection' is for something to look at itself (like in a mirror).
+
+Script reflection is a collection of powerful tools for scripts to inspect themselves.
+
+We can start with basic access to scripts.
+###
+
+
+example: # Getting a script
+
+	# You can obtain the current script
+	set {script} to the current script
+	{script} exists
+
+	# You can also find a script by name
+	set {script} to the script named "my cool script.sk"
+
+	# Even if it's in a folder
+	set {script} to the script named "examples/events.sk"
+
+	# Even if it's not enabled
+	set {script} to the script named "-disabled script.sk"
+
+
+example: # Looping scripts
+
+	# It's possible to find multiple scripts at a time.
+	set {scripts::*} to all loaded scripts
+	set {scripts::*} to to the scripts named "my script.sk" and "my cooler script.sk"
+	set {scripts::*} to the scripts in folder "examples/"
+
+	loop all loaded scripts:
+		broadcast name of loop-value
+
+
+example: # The name and path of a script
+
+	# Scripts have a name, which is their file name (without .sk)
+	set {script} to the script named "my script.sk"
+	the name of {script} is "my script"
+
+	# This doesn't include their folder name
+	set {script} to the script named "folder/my script.sk"
+	the name of {script} is "my script"
+
+	# However, if you just print out a script, it will include the whole path
+	set {script} to the script named "folder/my script.sk"
+	"%{script}%" is "folder/my script.sk"
+
+
+example: # Enabling, disabling and loading scripts
+
+	# You can enable a disabled script
+	set {script} to the script named "-disabled script.sk"
+	# This parses and loads its code
+	enable {script}
+
+	# Now that it's been enabled, you can disable it again
+	set {script} to the script named "disabled script.sk"
+	# This unloads its code and renames the script
+	disable {script}
+
+	# A disabled script is skipped when Skript loads
+	# If you want to remove it temporarily, you can `unload` it
+	set {script} to the script named "my cool script.sk"
+	unload {script}
+
+	# You can then load it again, which is the same as enabling
+	load {script}
+
+	# Or just reload it
+	reload {script}
+
+
+###
+CONFIGS
+=====
+
+There are also tools provided for reading Skript's built-in configuration files.
+These allow you to check values from these files.
+
+Configs are arranged as a tree of 'nodes', which have values:
+
+node name: value
+node name: value
```

Comment:
```suggestion
```

Reply:
It's supposed to be duplicated, it's an example that you can have multiple nodes at the same level

Reply:
ah this made it seem like you were describing the format of a node. maybe make the example use `regular node` like the rest?